.changeset/real-spoons-cough.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ultraviolet/form": minor
+---
+
+Prop `type` working properly
```

Comment:
```suggestion
Fix prop `type` on component `<TextInputFieldV2 />` that wasn't working properly
```

We try to add the component name so people will understand what change and where 😄